Modix.Data/Migrations/20200905130436_AddMessageContentPatterns.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+using Npgsql.EntityFrameworkCore.PostgreSQL.Metadata;
+
+namespace Modix.Data.Migrations
+{
+    public partial class AddMessageContentPatterns : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.CreateTable(
+                name: "MessageContentPatterns",
+                columns: table => new
+                {
+                    Id = table.Column<long>(nullable: false)
+                        .Annotation("Npgsql:ValueGenerationStrategy", NpgsqlValueGenerationStrategy.SerialColumn),
+                    Pattern = table.Column<string>(nullable: false),
+                    PatternType = table.Column<string>(nullable: false),
+                    GuildId = table.Column<long>(nullable: false)
+                },
+                constraints: table =>
+                {
+                    table.PrimaryKey("PK_MessageContentPatterns", x => x.Id);
+                });
+
+            migrationBuilder.Sql("UPDATE public.\"ClaimMappings\" SET \"Claim\"='BypassMessageContentPatternCheck' WHERE \"Claim\"='PostInviteLink';");
```

Comment:
If we're going to delete the invite purging behavior, I think we'll want to include the invite link regex as part of the migration here. Other guilds also use Modix, so I'd prefer not to break the invite link deletion functionality for them.